modMcf/src/org/aion/mcf/db/ContractDetailsCacheImpl.java

```diff
@@ -65,6 +65,7 @@ public IDataWord get(IDataWord key) {
                 return null;
             }
             value = origContract.get(key);
+            value = (value == null) ? DataWord.ZERO : value;
             storage.put(key.copy(), value.isZero() ? DataWord.ZERO.copy() : value.copy());
```

Comment:
DataWord.ZERO.copy() -> DataWord.ZERO

Reply:
looks like this PR need to merge the refine of the #573

Reply:
ah yes, this was just a quick temp fix for this branch, will update